.vimrc

```diff
@@ -8,6 +8,28 @@ call pathogen#infect()
 " disable vim-markdown folder
 let g:vim_markdown_folding_disabled=1
 
+" emmet
+let g:user_emmet_leader_key='<C-i>'
+
+" spotify
+let g:spotify_country_code = 'US'
+
+" remove trailing spaces
+command TRAIL %s/\s\+$//
+
+" html tidy - Meteor
+let g:syntastic_html_tidy_ignore_errors = [
+    \ '<form> lacks "action" attribute',
+    \ '<template> is not recognized!',
+    \ 'discarding unexpected <template>',
+    \ 'discarding unexpected </template>'
+  \ ]
+
+" gist-vim
+let g:gist_clip_command = 'pbcopy' " pbcopy for OSX Only
+let g:gist_detect_filetype = 1
+let g:gist_post_private = 1
```

Comment:
imo, these are pretty important settings for vim-gist. It makes your gists private, and copies the url to your "clipboard"/buffer.